draft-ietf-core-resource-directory.md

```diff
@@ -2509,6 +2509,45 @@ have been used to resolve the relative anchor values instead, giving
 
 and analogous records.
 
+## A note on differences between link-format and Link headers
+
+While link-format and Link headers look very similar and are based on the same
+model of typed links, there are some differences between {{RFC6690}} and
+{{RFC5988}} that should be kept in mind when using or implementing a Resource
+Directory:
+
+* There is no percent encoding in link-header documents.
+
+  A link-header document is an UTF-8 encoded string of Unicode characters and
+  does not have percent encoding, while Link headers are practically ASCII
+  strings that use percent encoding for non-ASCII characters, stating the
+  encoding explictly when required.
+
+  For example, while a Link header in a page about a Swedish city might read
+
+      Link: </temperature/Malm%C3%B6>;rel="live-environment-data";
+          title*=UTF-8'en'Live%20temperature%20from%20Malm%C3%B6;
+          other=fields
+
+  a link-format document from the same source might describe the link as
+
+      </temperature/Malmö>;rel="live-environment-data";
+      title*=UTF8'en'Live temperature from Malmö;other=fields
```

Comment:
We should fix that.  There should never have been title* target attributes in RFC 6690.

Reply:
... but not here -- I'm taking the `title*` part example out and'll leave that to 6690bis.